carlsim/test/stdp.cpp

```diff
@@ -469,7 +469,7 @@ TEST(STDP, ESTDPExpCurve) {
 
 	for (int mode = 0; mode < 2; mode++) {
 		for (int coba = 0; coba < 2; coba++) {
-			for (int offset = -30; offset <= 30; offset += 5) {
```

Comment:
Just a quick question: If this is built without GPU support, what happens when you run `GPU_MODE` (when `mode==1`)? Does it default to `CPU_MODE`? And if so, wouldn't that mean every test gets run twice?

If so, an easy way to cut STDPDAWeightBoost time in half would be to not run it only once in `CPU_MODE`.